docs/ref/contrib/sitemaps.txt

```diff
@@ -116,7 +116,7 @@ Note:
   attributes corresponding to ``<changefreq>`` and ``<priority>`` elements,
   respectively. They can be made callable as functions, as
   :attr:`~Sitemap.lastmod` was in the example.
-* :attr:`~Sitemap.items()` is a method that returns a list of objects. The
+* :attr:`~Sitemap.items()` is a method that returns a :term:`sequence` of objects. The
```

Comment:
I also wrote this in the ticket, but so the comment isn't missed when this PR is reviewed, I think `QuerySet` should be stated explicitly. The reason is that I think this language still leaves open the question of whether `QuerySet` objects are allowed (which is the reason I filed the ticket in the first place).

Reply:
I'm not sure if this is necessary, because this entire paragraph is about ["an example"](https://docs.djangoproject.com/en/dev/ref/contrib/sitemaps/#an-example) with `QuerySet`. That's why using `sequence` sounds sufficient to me.

Reply:
If this paragraph is about the example and the example uses a `QuerySet`, then I would just say `QuerySet` and not sequence. I think it's less clear to call a `QuerySet` a sequence when it can just be called what it is.

Regarding listing `QuerySet` as a possible return type, then my comment can be considered for below under `.. attribute:: Sitemap.items`. (I wasn't making a distinction between the two references when I first commented.)

Reply:
Agreed, we can mention `Queryset`, e.g.
```
A method that returns a :term:`sequence` of objects or a ``QuerySet``. 
```

Reply:
@Sjbrgsn Can you update your patch? :point_up: Please wrap also changed docs at 79 chars.